example-docs/fake_form_element/form.json

```diff
@@ -0,0 +1,149 @@
+[
```

Comment:
Is this a fixture for the expected output? I'd say put that under `exampledocs/test_evaluate_files/unstructured_output` since there are a few other expected JSON outputs in there.

Reply:
Gotcha, it looks like valid `unstructured` JSON output though right? In that case I'd still move it to the `unstructured_output` directory.

Reply:
Moving.